mail-client/alpine/alpine-2.21.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils flag-o-matic autotools multilib toolchain-funcs
+
+DESCRIPTION="alpine is an easy to use text-based based mail and news client"
+HOMEPAGE="http://www.washington.edu/alpine/ http://alpine.freeiz.com/alpine/ http://repo.or.cz/alpine.git/"
+SRC_URI="http://alpine.freeiz.com/alpine/release/src/${P}.tar.xz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc ipv6 kerberos ldap libressl nls onlyalpine passfile smime spell ssl threads"
+
+DEPEND="virtual/pam
+	>=sys-libs/ncurses-5.1:0=
+	ssl? (
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)
+	ldap? ( net-nds/openldap )
+	kerberos? ( app-crypt/mit-krb5 )
+	spell? ( app-text/aspell )
+"
+RDEPEND="${DEPEND}
+	app-misc/mime-types
+	!onlyalpine? ( !mail-client/pine )
+	!<=net-mail/uw-imap-2004g
+"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	local myconf="--without-tcl
+		--with-system-pinerc=/etc/pine.conf
+		--with-system-fixed-pinerc=/etc/pine.conf.fixed"
+		#--disable-debug"
+		# fixme
+		#   --with-system-mail-directory=DIR?
+
+	if use ssl; then
+		myconf+=" --with-ssl
+			--with-ssl-include-dir=/usr/include/openssl
+			--with-ssl-lib-dir=/usr/$(get_libdir)
+			--with-ssl-certs-dir=/etc/ssl/certs"
```

Comment:
all paths in general need `${EPREFIX}`, i.e. `--with-ssl-certs-dir="${EPREFIX}/etc/ssl/certs"`